src/writeData/subscriptions/components/BrokerDetails.tsx

```diff
@@ -0,0 +1,364 @@
+                      active={security === 'none'}
+                      onClick={() => {
+                        setSecurity('none')
+                      }}
+                      value="none"
+                      titleText="None"
+                      disabled={false}
+                    >
+                      None
+                    </SelectGroup.Option>
+                    <SelectGroup.Option
+                      name="user"
+                      id="user"
+                      testID="update-broker-form--user--button"
+                      active={security === 'user'}
+                      onClick={() => {
+                        setSecurity('user')
+                      }}
+                      value="user"
+                      titleText="User"
+                      disabled={false}
+                    >
+                      User
+                    </SelectGroup.Option>
+                    {/* For a later iteration */}
+                    {/* <SelectGroup.Option
+                      name="user"
+                      id="user"
+                      testID="user--button"
+                      active={security === 'certificate'}
+                      onClick={() => {
+                        setSecurity('certificate')
+                      }}
+                      value={'certificate'}
+                      titleText="Certificate"
+                      disabled={false}
+                    >
+                      Certificate
+                    </SelectGroup.Option> */}
+                  </SelectGroup>
+                  {security === 'user' && (
+                    <UserInput
+                      formContent={currentSubscription}
+                      updateForm={updateForm}
+                    />
+                  )}
+                </Grid.Column>
+              </Grid.Row>
+            </Grid>
+          </Overlay.Body>
+          <Overlay.Footer>
+            <Button
+              text="Close"
+              color={ComponentColor.Tertiary}
+              onClick={() => {
+                history.push(`/orgs/${org.id}/${LOAD_DATA}/${SUBSCRIPTIONS}`)
+              }}
+              titleText="Cancel update of Subscription and return to list"
+              type={ButtonType.Button}
+              testID="update-broker-form--cancel"
+            />
+            <Button
+              text="Edit"
+              color={edit ? ComponentColor.Success : ComponentColor.Secondary}
+              onClick={() => setEdit(!edit)}
+              type={ButtonType.Button}
+              titleText="Edit"
+              testID="update-broker-form--submit"
+            />
+            <Button
+              text="Next"
+              color={ComponentColor.Secondary}
+              onClick={() => setFormActive('subscription')}
+              type={ButtonType.Button}
+              titleText="Next"
+              testID="update-broker-form--submit"
+            />
+            <Button
```

Comment:
right now this just forwards them to notebooks not sure if we wanted more than that